app/views/volunteers/show.html.slim

```diff
@@ -2,6 +2,11 @@
   .col-xs-12
     h1= @volunteer.contact.full_name
 
+- if Volunteer.seeking_clients.include?(@volunteer)
```

Comment:
Doing DB queries from a view should usually be avoided, and it's also not necessary here since you already have the volunteer's `state`. It's not a big deal, but you could e.g. add a `Volunteer::SEEKING_STATES` constant, use it in `Volunteer.seeking_clients`, add an instance method `Volunteer#seeking_clients?`, and then just do `@volunteer.seeking_clients?` here.